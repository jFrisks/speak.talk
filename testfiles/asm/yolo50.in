A function that is called fac and uses the arguments of an integer that is called x. It returns an integer by running the following code:
  - if the variable that is called x is less or equal to 1, do the following:
    - finally return 1;
  .
  - finally return the value of the function fac that uses the arguments of the variable that is called x subtracted by1 multiplied with x;
.
A function that is called binom and uses the arguments of an integer that is called n, an integer that is called k. It returns an integer by running the following code:
  - if the variable that is called n is equal to1, do the following:
    - finally return 1;
  .
  - if the variable that is called n is equal tok, do the following:
    - finally return 1;
  .
  - if the variable that is called k is greater thann, do the following:
    - finally return 0;
  .
  - finally return the value of the function fac that uses the arguments of the variable that is called n / (the value of the function fac that uses the arguments of the variable that is called k multiplied with the value of the function fac that uses the arguments of the variable that is called n subtracted by the variable that is called k);
.
A function that is called main. It returns an integer by running the following code:
  - run the function that is called print and uses the arguments of the value of the function binom that uses the arguments of 12, 3;
  - run the function that is called print and uses the arguments of the value of the function binom that uses the arguments of 12, 4;
  - run the function that is called print and uses the arguments of the value of the function binom that uses the arguments of 12, 5;
  - run the function that is called print and uses the arguments of the value of the function binom that uses the arguments of 12, 6;
  - finally return 0;
.