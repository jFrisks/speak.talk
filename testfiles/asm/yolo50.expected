220
495
792
924
